### DISCUSSION TITLE
How to you set host and port numbers?

### DISCUSSION THREAD

**Discussion by a1270** [chrisbenincasa/tunarr#734] [Questions & Help] [answered]
I have tried all the normal flags to set host and port and nothing. Searching the repo i can't find any reference to any conf i can edit either.

**Comment by chrisbenincasa** [comment, marked as answer]
I'm assuming you are running the binary directly, in which case, you can use the `--port` argument for `TUNARR_SERVER_PORT` environment variable to set the port. For bind address you can use the `TUNARR_BIND_ADDR` environment variable. Some examples for the macOS 'binary':

```
➜  tunarr-macos-x64 ) ./macos-entrypoint.sh -- --port 1234 | grep port
2024-08-29T20:25:49.944Z [info]: HTTP server running on port: http://localhost:1234
➜  tunarr-macos-x64 ) TUNARR_SERVER_PORT=1234 ./macos-entrypoint.sh | grep port
2024-08-29T20:25:58.227Z [info]: HTTP server running on port: http://localhost:1234
➜  tunarr-macos-x64 ) TUNARR_BIND_ADDR=0.0.0.0 ./macos-entrypoint.sh | grep port
2024-08-29T20:26:10.141Z [info]: HTTP server running on port: http://0.0.0.0:8000
```

If you are running in docker, you generally do not want to change the bind address (unless you are running in `host` network mode). Ports can be specified using Docker port mappings rather than changing the port that Tunarr binds within the container.
